[2023-10-13 19:10:34,265] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: consultar_info_covid_api.load_data_redshift scheduled__2023-10-12T00:00:00+00:00 [queued]>
[2023-10-13 19:10:34,271] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: consultar_info_covid_api.load_data_redshift scheduled__2023-10-12T00:00:00+00:00 [queued]>
[2023-10-13 19:10:34,272] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-10-13 19:10:34,272] {taskinstance.py:1377} INFO - Starting attempt 9 of 14
[2023-10-13 19:10:34,272] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-10-13 19:10:34,284] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): load_data_redshift> on 2023-10-12 00:00:00+00:00
[2023-10-13 19:10:34,287] {standard_task_runner.py:52} INFO - Started process 194 to run task
[2023-10-13 19:10:34,289] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'consultar_info_covid_api', 'load_data_redshift', 'scheduled__2023-10-12T00:00:00+00:00', '--job-id', '379', '--raw', '--subdir', 'DAGS_FOLDER/api_covid.py', '--cfg-path', '/tmp/tmp8v81eat4', '--error-file', '/tmp/tmp6hg6ztcp']
[2023-10-13 19:10:34,289] {standard_task_runner.py:80} INFO - Job 379: Subtask load_data_redshift
[2023-10-13 19:10:34,329] {task_command.py:371} INFO - Running <TaskInstance: consultar_info_covid_api.load_data_redshift scheduled__2023-10-12T00:00:00+00:00 [running]> on host 738c88afe20a
[2023-10-13 19:10:34,378] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Axel Villegas Luna
AIRFLOW_CTX_DAG_ID=consultar_info_covid_api
AIRFLOW_CTX_TASK_ID=load_data_redshift
AIRFLOW_CTX_EXECUTION_DATE=2023-10-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=9
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-12T00:00:00+00:00
[2023-10-13 19:11:34,227] {logging_mixin.py:115} INFO - Exito
[2023-10-13 19:11:34,228] {python.py:173} INFO - Done. Returned value was: None
[2023-10-13 19:11:34,238] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=consultar_info_covid_api, task_id=load_data_redshift, execution_date=20231012T000000, start_date=20231013T191034, end_date=20231013T191134
[2023-10-13 19:11:34,266] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-13 19:11:34,285] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
